.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Proposal
+about: Propose an addition to the spec
+title: "[PROPOSAL]"
+labels: proposal
+assignees: ''
+
+---
+
+**Purpose:**
+This section gives the context of the proposal. It explains *why* this is needed
+Please describe the corresponding use cases.
```

Comment:
_Minor_: You may want to add a `.` after "...this is needed"? When the markdown is rendered, a new line isn't present.

Reply:
thank you.